### ISSUE TITLE
Question: Are the trained agents being evaluated on separate test env? 

### ISSUE THREAD

**Issue by vihangp** [joonleesky/train-procgen-pytorch#1]
Hi,

I had a question regarding the evaluation of the performance of the agent during training. Is the agent evaluated on separate test envs (with different levels)? I am not able to find the code where this evaluation is happening. 

In case you are evaluating on a separate test set, which levels are you evaluating on?
